catalog-shacl.shce

```diff
@@ -28,14 +28,14 @@ shape :CreativeWorkShape -> ex:CreativeWork ;
 		sh:name "landing page"@en ;
 		sh:description "URL(s) where the work is available, or described"
 	% .
-	ex:provider IRI %
```

Comment:
Is this intentional - it doesn't look like `OrganizationShape` requires focus nodes to be IRIs.

Reply:
They would be redundant. I suggest confirming this by running validation eith a test case you expect to fail.